Kajabi/sage-lib
# [Sage-8] Tokens - Font Family Update to Circular

## Description

Updates the Sage font family from Inter to Circular.

Note: Linting is failing because the fonts aren't being committed.

## Screenshots

|  Before  |  After  |
|--------|--------|
<img width="960" alt="Screen Shot 2022-01-27 at 11 19 04 AM" src="https://user-images.githubusercontent.com/14791307/151411335-aa1be933-6a35-4e3d-97fc-e450a7446ca5.png">|<img width="948" alt="Screen Shot 2022-01-27 at 11 18 28 AM" src="https://user-images.githubusercontent.com/14791307/151411353-bb33603d-0da0-4dea-b4b0-399a054530e8.png">


## Testing in `sage-lib`

- View Sage Docs
- Check that new font, Circular, is properly in use.

## Testing in `kajabi-products`

(LOW) Updates the Sage font family from Inter to Circular. No effect on Kajabi Products.


## Related

[Sage-8](https://kajabi.atlassian.net/browse/SAGE-8)
